### PR TITLE
Add cluster config command

### PR DESCRIPTION
**Pull request by colin014**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Support to download K8S config to the current folder.

### Usage
```
Get K8S config

Usage:
  banzai cluster config [--cluster=ID | [--cluster-name=]NAME] [flags]

Aliases:
  config, co

Flags:
      --cluster int32         ID of cluster to config
      --cluster-name string   Name of cluster to config
  -h, --help                  help for config
  -p, --path string           Path to save cluster K8S config

Global Flags:
      --color                use colors on non-tty outputs
      --config string        config file (default is $BANZAICONFIG or $HOME/.banzai/config.yaml)
      --interactive          ask questions interactively even if stdin or stdout is non-tty
      --no-color             never display color output
      --no-interactive       never ask questions interactively
      --organization int32   organization id
  -o, --output string        output format (default|yaml|json) (default "default")
      --verbose              more verbose output
```

### Checklist

- [x] Error handling code meets the [guideline](https://github.com/banzaicloud/pipeline/blob/master/docs/error-handling-guide.md)
- [x] Logging code meets the guideline (TODO)
- [x] User guide and development docs updated (if needed)

### CODE REVIEW

File: internal/cli/command/cluster/config.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"emperror.dev/errors"
+	"github.com/banzaicloud/banzai-cli/internal/cli"
+	clustercontext "github.com/banzaicloud/banzai-cli/internal/cli/command/cluster/context"
```
Comment (Laci21) [inline internal/cli/command/cluster/config.go:24]: I guess these imports should be lower separately.

Comment (colin014) [reply internal/cli/command/cluster/config.go]: thx @Laci21

**Comment by Laci21** [comment]
Shouldn't docs be generated in this repo?

Also, I don't want to be insatiable but wouldn't it be possible to implement a way so that users can provide a download path as well? :)

**Comment by colin014** [comment]
The docs removed from this repo and moved under banzaicloud.com (https://banzaicloud.com/docs/pipeline/cli/reference/)

The path is a good idea, I'll add later

**Pull request merged**